### PR TITLE
boards/arduino-common: export led via saul

### PR DESCRIPTION
**Pull request by mali**
expose on-board led to saul interface.

### CODE REVIEW

File: boards/arduino-common/include/gpio_params.h [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   boards_boards_arduino-common
```
Comment (aabadie) [inline boards/arduino-common/include/gpio_params.h]: boards_arduino-common ?

**Comment by miri64** [comment]
Postponed due to feature freeze.

**Comment by mali** [comment]
@aabadie , I've just copied what has been done for other boards ;-)
I need to learn a bit more about saul and how to test this

**Comment by PeterKietzmann** [comment]
As far as I see it `arduino-common` is just used by arduino-duemilanove and arduino-uno boards. @aabadie which one did you try? Code-wise the PR looks fine. But I have no device to test... @kYc0o do you?

**Comment by kYc0o** [comment]
I can do it I have a board ;-)

**Comment by PeterKietzmann** [comment]
@kYc0o what was the outcome?

**Comment by kYc0o** [comment]
The default example was working by chance with almost full memory... With this change the memory is full and the board gets stuck...

**Comment by kYc0o** [comment]
Well... actually not even the default example in current master can run on any "arduino-common" board (BTW we should change this name, it doesn't include all the arduino boards but the smaller ones).

So at least with current tests/examples this PR cannot be tested (tests/saul prints only `INFO # Data:	[0] 0` very fast... xtimer problem?).

How did you test this @mali ?

**Comment by mali** [comment]
@kYc0o, I've just built tests/saul/ 
as you said, it prints `INFO # Data:  [0] 0` while looping fast.
I thougt it was OK, as the test performs a saul_reg_read() and here dev[0] is a led.
Anyway, if it doesn't fit in current master, i think this PR should be discarded.

**Comment by kYc0o** [comment]
Yes that's what I see... I think the PR is OK, but we need a way to use it and usually (I think) we can do it with default.

I'd ACK but @aabadie requested changes so I'll let him validate it.

Please squash and let's run Murdock.

@haukepetersen is it SAUL intended for other uses than user interaction?

**Comment by kYc0o** [comment]
Ah! I almost forget, can you open an issue about the xtimer? (Related to the fast printing)

**Comment by mali** [comment]
errors in cortex_mx_y build of tests/pkg_cmsis-dsp/ seems unrelated... can it be ?

**Comment by PeterKietzmann** [comment]
Yes, it's unrelated. See #6356. Let's wait for that PR (@gebart's review required), then rebase and merge

**Comment by aabadie** [comment]
> errors in cortex_mx_y build of tests/pkg_cmsis-dsp/ seems unrelated... can it be ?

Errors are unrelated. I have the same on other PRs with Murdock. @kaspar030 do you have an idea what's the problem ?

**Comment by PeterKietzmann** [comment]
@aabadie see my comment above

**Comment by aabadie** [comment]
@mali, #6356, has been merged. Can you rebase ?

**Comment by aabadie** [comment]
All green : Go!
Thanks @mali !

**Pull request merged**